demo/screen.py

```
@@ -0,0 +1,107 @@
+
+import sys
+from abc import ABC
+
+import pygame
+import pygame.locals
+
+from demo.colors import Colors
+from demo.resources import Resources
+
+
+class Screen(ABC):
```

Ruff (0.12.2)

`Screen` is an abstract base class, but it has no abstract methods or properties

(B024)